test-crates/duchess-java-tests/tests/ui/examples/auth.rs

```diff
@@ -1,4 +1,4 @@
-//@check-pass
+//@run
```

Comment:
Once all tests are in `run` mode, we can switch them globally and remove all the annotations